server/controllers/admin/employees.js

```diff
@@ -190,43 +216,43 @@ exports.update = function update(req, res, next) {
     employee.date_embauche = new Date(employee.date_embauche);
   }
 
-  db.exec(sql, [employee, req.params.id])
-  .then(function (row) {
+  delete(employee.debitor_group_uuid);
```

Comment:
Also,  is there any way that you can do a lot of this triage on the client in the EmployeeService as well?